### DISCUSSION TITLE
Seal of Vengeance at lvl 64 Seal of Corruption at lvl 66.change to same lvl?

### DISCUSSION THREAD

**Discussion by Tituspullo666** [chromiecraft/chromiecraft#4458] [Suggestions]
Alliance paladins get Seal of Vengeance at lvl 64 and horde gets Seal of Corruption at lvl 66 currently on Chromiecraft. Posted it in bugreports and got the reply that this is blizzlike for Wrath retail. My suggestion is to make Seal of Corruption available at lvl 64 since the 64 bracket will last nearly 2 months and Horde paladins would be clearly disadvantaged during this period. PvP has to be considered during that bracket also; same class but 1 faction has single target dot and higher single target damage potential.

**Comment by sombritas** [comment]
i also feel like keeping something blizzlike for the sake of blizzlike in this bracket division is somewhat out of place. i would also prefer that both factions get the skill at the same time at least, sure, at 70 everyone gets their skills but since the bracket division is arbitrary, it doesn't feel right to favour one side for 2 months just because of that

**Comment by Jokkbrock** [comment]
As a horde paladin main myself, I agree. I should not feel the urge to faction change if I want to play Ret at the highest lvl next bracket. 

This is an example where the "it's blizzlike"-argument doesnt really work. For example doesnt Druids get flight form at lvl 60 on Wrath retail? But here they will get flight with everyone else at lvl 70 (I assume since flying is changed on this server). And that change is ok, but this change isnt?

And level cap at lvl 64 isnt Blizzlike either, so just give both factions their new seal at the same time. To not make it unbalanced over the factions, when everything else class-wise is the same.

**Comment by expigator** [comment]
Agreed with above based solely on CC release brackets

**Comment by heyitsbench** [comment]
Could also delay Seal of Vengeance to the 65-69 bracket, similar to flying.

**Comment by Peti321** [comment]
I'm not even a paladin but I think its not fair. I know its basically Blizzlike but Blizzlike has no brackets. Make the Vengeance unlock at level 66.

**Comment by FFabz** [comment]
As a horde ret I really don't care, at 64 we'll stomp every dungeon with vanilla raid gear, bosses will be down fast at a point where there will be almost no diff in dps between vengence and command, same for pvp as ret you can one shot most classes with one rotation (with raid gear), so only a point against tank classes whom as ret we cannot kill in 1V1 anyway (most of the time). Only reason you want it is to min max a lvl 64 instance... which we will already obliterate with AQ40 and BWL gear, yes it's not "fair" i guess but it won't change anything

**Comment by Amrx2** [comment]
Hello, Paladins! So you will get this news also via an official post but don't worry, we are correcting this issue created mainly due to our bracket system (and Blizzard giving factions different skills at different times). You will both get Vengence at the same time!

Cheers,
Construct
